### PR TITLE
Adding external resources links

### PR DESCRIPTION
**Pull request by mishuagopian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,13 +35,33 @@
...
 
-### Carousel
-
-### Modal
-
-### Parallax
-
-### Forms Error
+### Images Presentation
+  - Multi Layout Slideshow
```
Comment (sbalay) [inline README.md]: I'd list these as links
```
- [Multi Layout Slideshow](http://tympanus.net/Development/MultiLayoutSlideshow/)
- [Image Grid Effects](http://tympanus.net/Development/ImageGridEffects/)
```

Comment (mishuagopian) [inline README.md]: Ok!

**Comment by sbalay** [comment]
👍

**Comment by SKOLZ** [comment]
👍

**Pull request merged**